extensions/extensions-helm/aissemble-jenkins-chart/README.md

```diff
@@ -14,13 +14,13 @@ _**NOTE:**_ *the version should match the aiSSEMBLE project version.*
 See the official Helm chart [documentation](https://github.com/jenkinsci/helm-charts/tree/main/charts/jenkins). The following
 default values have been modified from the original helm chart.
 
-| Property                 | Default                                                                              |
-|--------------------------|--------------------------------------------------------------------------------------|
-| `controller.image`       | NB: OSS: <default aissemble docker repo>/boozallen/aissemble-jenkins-controller      |
-| `controller.tag`         | _Chart.Version_                                                                      |
-| `controller.testEnabled` | false                                                                                |
-| `agent.image`            | NB: OSS: <default aissemble docker repo>//boozallen/aissemble-jenkins-agent |
-| `agent.tag`              | _Chart.Version_                                                                      |
+| Property                 | Default                                                                       |
+|--------------------------|-------------------------------------------------------------------------------|
+| `controller.image`       | docker-registry-PLACEHOLDER/repository/boozallen/aissemble-jenkins-controller |
+| `controller.tag`         | _Chart.Version_                                                               |
+| `controller.testEnabled` | false                                                                         |
+| `agent.image`            | NB: OSS: <default aissemble docker repo>//boozallen/aissemble-jenkins-agent   |
```

Comment:
A: Let's go ahead and update this while we're making changes.

Reply:
Updated as well ✅